src/index.ts

```diff
@@ -65,6 +65,7 @@ function fixDeprecatedOptions(options: any): ProjectOptions {
 export interface ProjectBuildOptions {
   name?: string;
   addServiceWorker?: boolean;
+  addPushManifest?: boolean;
```

Comment:
The IDE uses this interface to provide inline documentation in polymer.json. I think it'd be super helpful to add docs to as many of these properties as possible.